### PR TITLE
Enable pip check

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
Fixes https://github.com/conda-forge/pytorch-lightning-feedstock/issues/114

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @borda @carmocca @williamFalcon
\ No newline at end of file
+* @borda @carmocca @jan-janssen @williamFalcon
```
Comment (Borda) [inline .github/CODEOWNERS:1]:
```suggestion
* @borda @carmocca @williamFalcon
```

**Review by Borda — CHANGES_REQUESTED** [review]
I would split adding maintainer from fixing an issue, pls

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
> I would split adding maintainer from fixing an issue, pls

Sure - no pressure. I just wanted to help out.

**Review by Borda — APPROVED** [review]

**Pull request merged**